### PR TITLE
 Fix use of trailing */ in dox, which messes with sipify Python docs

### PR DESCRIPTION
**Pull request by nyalldawson**
Makes it looks like there's a * footnote for a docstring line.

### CODE REVIEW

**Comment by nyalldawson** [comment]
Unrelated test failure

File: src/core/qgspluginlayerregistry.h [hunk 1]
```diff
@@ -82,7 +82,7 @@ class CORE_EXPORT QgsPluginLayerRegistry
 
     /**
      * List all known layer types
-     *  \since QGIS */
+     */
```
Comment (DelazJ) [inline src/core/qgspluginlayerregistry.h]: @nyalldawson just me curious: looking to the changes, should this be aligned with line 83?

Comment (nyalldawson) [reply src/core/qgspluginlayerregistry.h]: There's no consistency there yet -- maybe we need a new script to fix that :)

Comment (3nids) [reply src/core/qgspluginlayerregistry.h]: I was wondering if we should not drop astyle in favor of clang-format. Anyone has an opinion here?

**Pull request merged**